plugins/handlers/gallery.py

```diff
@@ -251,27 +267,85 @@ def evaluate(self, data):
         return False
 
 
+class And(object):
+
+    def __init__(self, *tests):
+        self.tests = tests
+
+    def evaluate(self, data):
+        for test in self.tests:
+            if not test.evaluate(data):
+                return False
+        return True
+
+
 class Glob(Or):
 
     def __init__(self, glob):
         patterns = [fnmatch.translate(p) for p in braceexpand.braceexpand(glob)]
         tests = [Regex(pattern, re.IGNORECASE) for pattern in patterns]
         super().__init__(*tests)
 
+    def evaluate(self, data):
+        return super().evaluate(data["basename"])
+
+
+# TODO: Consider making this an explicit metadata key test.
```

Comment:
Not sure what the best approach here would be yet. It would be nice to be more explicit, but the code will end up looking more bloated.